tasks/eva_2038/storeSSIDsInVariantWarehouse.py

```diff
@@ -0,0 +1,114 @@
+import argparse
+import logging
+import os
+import random
+import shutil
+import string
+import subprocess
+
+import yaml
+from ebi_eva_common_pyutils import command_utils
+from ebi_eva_common_pyutils.logger import logging_config as log_cfg
+from ebi_eva_internal_pyutils.metadata_utils import get_metadata_connection_handle, resolve_variant_warehouse_db_name
+from ebi_eva_internal_pyutils.pg_utils import get_all_results_for_query
+
+logger = log_cfg.get_logger(__name__)
+log_cfg.add_stdout_handler()
+log_cfg.set_log_level(logging.INFO)
+
+
+def run_nextflow(working_dir, params, project_id, accession_report_file, db_name):
+    random_string = ''.join(random.choice(string.ascii_letters) for i in range(6))
+    workflow_name = f"import_accession_{project_id}_{random_string}"
```

Comment:
Updated.
Since we might have to run multiple jobs for multiple accession report files present in the same project, this was just to give each one a separate work directory under Project.
You are right using the accession report name would be more useful. have updated it.